src/debugger/sourceMap.ts

```diff
@@ -15,7 +15,7 @@ interface ISourceMap {
 }
 
 export class SourceMapUtil {
-    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+?)\s*$/m;
+    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+index\.ios\.map.*?)\s*$/m;
```

Comment:
Ok, you're right. I'll investigate why we have multiple source maps in one body (maybe it's related to babel telling me ```[BABEL] Note: The code generator has deoptimised the styling of "/Users/mark/**/node_modules/immutable/dist/immutable.js" as it exceeds the max of "100KB".``` and remove this change from the PR